ui/shared/status/StatusLoadingIndicator.qml

```diff
@@ -1,14 +1,13 @@
 import QtQuick 2.13
 import "."
 
-SVGImage {
-    id: loadingImg
-    source: "../app/img/loading.svg"
-    width: 17
+StatusIcon {
+    id: root
```

Comment:
i know than all qml examples use `id: root`, but for many library components inner id is the same as file or i use `id: __impl`. Double underscore is old hack for telling than its a local id.

Reply:
> but for many library components inner id is the same as file

That's interesting! I don't have strong opinions how we name the root of a component. However we should stay consistent, which means, we'll have to touch many existing components that by now use `root`.